utils/aoa-proxy/Makefile

```
@@ -0,0 +1,62 @@
+#
+# Copyright (C) 2023 Jó Ágila Bitsch
+#
+# SPDX-License-Identifier: GPL-2.0-only
+
+include $(TOPDIR)/rules.mk
+ 
+PKG_NAME:=aoa-proxy
+PKG_VERSION:=0.2.2
+PKG_RELEASE:=1
+ 
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/jo-bitsch/aoa-proxy/archive/refs/tags/v$(PKG_VERSION).tar.gz?
+PKG_HASH:=28501cacd03d274077428c69edbbdc54855be68b77ea7d8359895009889d2631
+
+PKG_BUILD_DEPENDS:=!USE_GLIBC:argp-standalone +libb64
+
+PKG_MAINTAINER:=Jó Ágila Bitsch <jgilab@gmail.com>
+PKG_LICENSE:=BSD-3-Clause
+PKG_LICENSE_FILES:=LICENSE
+ 
+include $(INCLUDE_DIR)/package.mk
+
+define Package/aoa-proxy
+  SECTION:=utils
+  CATEGORY:=Utilities
+  TITLE:=Interact with Android devices using AOA
+  URL:=https://github.com/jo-bitsch/aoa-proxy
+  DEPENDS:=+libusb-1.0
+endef
+ 
+define Package/aoa-proxy/description
+  This program aims to make working with Android Open Accessory (in 
+  particular v1) as easy as possible.
+  It has 2 modes, depending on the state of the attached Android device.
+  (1) Announce mode: When the Android device is not yet in AOA mode, this
+      program announces its identity to the Android device and tries to
+      put it into AOA mode.
+  (2) Forwarding mode: Forward all input from stdin to the AOA device and
+      all output from AOA to stdout.
+  For OpenWrt, it includes a hotplug script and a service to automatically
+  announce itself and forward AOA traffic to SSH/dropbear.
+endef
+
+define Build/Compile
+	$(MAKE) -C $(PKG_BUILD_DIR) \
+		$(TARGET_CONFIGURE_OPTS) \
+    GIT_VERSION="v$(PKG_VERSION)" \
+		CFLAGS="$(TARGET_CFLAGS)" \
+		LDFLAGS="$(TARGET_LDFLAGS)"
+endef
```

Can't you do something like:
```suggestion
MAKE_FLAGS+= \
	GIT_VERSION="v$(PKG_VERSION)"
```